sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -5,13 +5,16 @@
 ### Features Added
 
 - Add support for session-level sanitization using the `Recorder.addSessionSanitizers` static method. [#21533](https://github.com/Azure/azure-sdk-for-js/pull/21533)
```

Comment:
Update version to 2.1.0?

Reply:
I tried before but the changelog validation didn't like it very much. I can give it another go